Reorder README types in constants for consistency

kadiyam229 opened the pull request:
## Hey, I just made a Pull Request!

Updated the [README_TYPES] array to look for README.md first. 

![image](https://github.com/user-attachments/assets/8adfdfee-85a2-4006-b238-da9d7d76a720)


### Context

The plugin does a call to bitbucket looking for each filename in its list for a README file. I think the list has five names, and README.md is last, so there are five requests to get a README file every time which is causing latency issue. 

### Issue ticket number and link

- Fixes # (issue)

### Checklist before requesting a review

- [* ] I have performed a self-review of my own code
- [ ] I have verified that the code builds perfectly fine on my local system
- [ ] I have verified that my code follows the style already available in the repository
- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/AxisCommunications/backstage-plugins/blob/main/CONTRIBUTING.md#changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ *] Screenshots attached (for UI changes)

anicke commented:
Makes sense. I have created the related issue #258, hopefully that could be fixed soon.

kadiyam229 commented:
Can someone hep me to understand the failurehere

anicke commented:
Have you tried to run "yarn prettier:fix"? That should fix it.

kadiyam229 commented:
> Have you tried to run "yarn prettier:fix"? That should fix it.

Thanks, how do I get this approved and merged?

anicke commented:
Hi! Do you need any help with this? I rebased this and added some tests and a changeset here:

https://github.com/AxisCommunications/backstage-plugins/compare/main...Reorder-README

kadiyam229 commented:
> Hi! Do you need any help with this? I rebased this and added some tests and a changeset here:
> 
> [main...Reorder-README](https://github.com/AxisCommunications/backstage-plugins/compare/main...Reorder-README)

Hi, thanks for letting me know. I am don't see any issues in my local but the workflow seems failing. Can you please check from your side and feel free to update if required.

kadiyam229 commented:
> I did some updates in #273, we can continue there?

Great! appreciated

anicke commented:
If you update  "@axis-backstage/plugin-readme-backend"  to 0.13.0 should be able configure the "README" files handling.
